website/ref/math.md

```diff
@@ -0,0 +1,11 @@
+<!-- toc -->
+
+# Introduction
+
+The `math:` module provides mathematical functions and constants.
+
+Function usages are given in the same format as in the reference doc for the
+[builtin module](builtin.html). In particular, the discussion of [Commands
```

Comment:
> In particular, ...

* Write complete sentences to make the meaning unambiguous.

* Only use title case when explicitly referring to a section (such as "see section Number Type"). In this case you are referring to the *content* of a section.

In this case, write:

`In particular, all the commands in this module conform to the pattern of [commands that operate on numbers](...).`